proto/cosmos/accesscontrol/accesscontrol.proto

```diff
@@ -28,9 +33,9 @@ message MessageDependencyMapping {
 
 message WasmDependencyMapping {
     bool enabled = 1;
```

Comment:
I think we can probably deprecate the enabled actually, since we don't meaningfully use it and maintaining behavior around it is additional effort that's not necesarily worth it. The main use case would be for resetting wasm dependencies, but we've been leaving it enabled but with synchronous dependencies instead. What we could do instead is simply reset it to synchronous or even delete the wasm dependency to "reset" it aka clear out unneeded store state and rely on the behavior that returns Synchronous access ops when no registered dependencies exist.
wdyt @codchen

Reply:
yeah i agree. Removed